ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -38,18 +38,18 @@ LT_DC_SC:
   - DC 
   - SC
 
-#Noncanonicl types for Delmatic lighting controller at AU-SYD-ODI
+#Noncanonicl types
 LGRP_DC_MS_IM:
-  description: "Delamtic controller lighting groups with brightness, motion sensing and illuminance monitoring"
+  description: "Non-canonical lighting groups with brightness, motion sensing and illuminance monitoring"
   is_canonical: false
   implements:
   - LGRP
   - DC
   - MS
   - IM
 
-ELT_DELMATIC:
-  description: "Delamtic emergency light"
+ELT_GENERIC:
```

Comment:
Should this have a new general type `ELT` added? If so please add to general types file.

Reply:
Added ELT type to generaltypes and updated to implement here.